README.md

```diff
@@ -50,25 +50,73 @@ You should start 4 instances parallel.
 Instance 1
 
 ```
+First run
+brew install librdkafka
+
+then
+make install
+
+To view brew services to make sure mosquitto has started
+brew services list
+
+then run
+git branch
+git log
+git pull
+
+then
 mosquitto -c config/mosquitto.conf
+
+and
+brew services start mosquitto
+ 
 ```
 
 Instance 2
 
 ```
+cd ssa_iot
+
```

Comment:
also please remove the line brake for consistency. Please apply to all the code section.